packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -287,6 +304,31 @@ export class MonitorATMServicesViewImpl extends React.PureComponent<TPropsWithIn
                 ))}
               </Field>
             </Col>
+            <Col span={6}>
+              <h2 className="span-kind-selector-header">Choose kind</h2>
```

Comment:
```suggestion
              <h2 className="span-kind-selector-header">Span Kind</h2>
```
Similarly, `s/Choose service/Service/` in L287, using the word "choose" in the UX is redundant

Reply:
Removed "Choose" in: [07e3c26](https://github.com/jaegertracing/jaeger-ui/pull/2012/commits/07e3c26244e78643a3d194cf38693988d2e7639c)